Tasks/RenderTemplatesTask.php

```diff
@@ -15,7 +15,7 @@ class RenderTemplatesTask extends Task
 
     public function __construct()
     {
-        $this->templatePath = 'Containers/' . config('vendor-documentation.section_name') . '/Documentation/ApiDocJs/shared/header.template.md';
+        $this->templatePath = 'Containers/' . config('vendor-documentation.section_name') . '/Documentation/ApiDocJs/shared/header.template'.config('vendor-documentation.locale', '').'.md';
```

Comment:
Please change the config to use the `en` header by default
`config('vendor-documentation.locale', 'en')`
and also change the default header to `header.template.en.md`

Reply:
Done! :)